ansible_collections/arista/avd/roles/eos_designs/defaults/main/main.yml

```diff
@@ -121,6 +121,9 @@ connected_endpoints_keys:
   storage_arrays:
     type: storage_array
 
+tenants_keys:
```

Comment:
sorry I didn't catch this before. Can you make it `network_services_keys` ? Thanks.